### PR TITLE
fix: 🐛 support bigger images

### PR DESCRIPTION
**Pull request by severo**
fixes https://github.com/huggingface/datasets-server/issues/1361

### CODE REVIEW

File: services/worker/src/worker/utils.py [hunk 1]
```diff
@@ -33,6 +34,9 @@
 from libcommon.exceptions import NormalRowsError, StreamingRowsError
 from libcommon.utils import orjson_dumps
 
+MAX_IMAGE_PIXELS = 1_000_000_000
```
Comment (albertvillanova) [inline services/worker/src/worker/utils.py:37]: Do you think it is good to have this constant here instead of in a config file?

**Review by albertvillanova — APPROVED** [review]
Thanks!
Just wondering if we can properly handle this larger size...

File: services/worker/src/worker/utils.py (hunk 1 above)
Comment (severo) [reply services/worker/src/worker/utils.py:37]: I was wondering if we really want to configure this, or if it's OK to hardcode. For now, I would say it's OK. Maybe we could move it to a constants.py file if it's cleaner (I think it's not the only hardcoded value not in a constants.py file).

Comment (lhoestq) [reply services/worker/src/worker/utils.py:37]: ok to have it here imo, but if it's used somewhere else in the future we can move it

**Review by lhoestq — APPROVED** [review]
cool !

**Pull request merged**